src/CMakeLists.txt

```diff
@@ -1,8 +1,19 @@
 find_package(PkgConfig REQUIRED)
-pkg_check_modules(EventdEvent REQUIRED libeventd-event>=0.3)
+pkg_check_modules(EventdEvent REQUIRED libeventd>=0.3)
 pkg_check_modules(EventdPlugin REQUIRED libeventd-plugin>=0.3)
 pkg_check_modules(SystemdJournal REQUIRED libsystemd-journal)
 
+macro(pkg_config_variable PKG VAR OUT)
+    execute_process(COMMAND ${PKG_CONFIG_EXECUTABLE} --variable=${VAR} ${PKG}
+        OUTPUT_VARIABLE ${OUT})
+    string(STRIP ${${OUT}} ${OUT})
+endmacro(pkg_config_variable)
+
+pkg_config_variable(libeventd pkglibdir EVENTD_PKGLIBDIR_INTERNAL)
+find_library(EVENTDHELPERS_LIBRARY eventd-helpers
```

Comment:
This is an internal helpers library. The fact that you are using it *is* the exception. I do not want to maintain that API in any stable way. :-)